### PR TITLE
completed signup page

### PR DESCRIPTION
**Pull request by Adarsh-Patel-Dev**
image

### CODE REVIEW

**Comment by swstica** [comment]
The nav items on right are not displaying to the extreme right on my screen.
This is how it's looking.

Capture11

File: index.css [hunk 1]
```diff
@@ -0,0 +1,241 @@
...
+.nav--search--input {
+    border-radius: 1rem;
+    border: none;
+    padding: 0.5rem;
+    width: 40rem;
+}
+
+.nav--search--input {
+    /* position: absolute; */
+    /* width: ; */
```
Comment (sri-rishi) [inline index.css]: remove comments

File: index.css [hunk 2]
```diff
@@ -0,0 +1,241 @@
...
+    height: 3rem;
+    padding: 2rem;
+    border: 1px solid gray;
+    background: rgba(255, 255, 255, 0.11);
```
Comment (sri-rishi) [inline index.css:55]: use variable color naming

File: index.css [hunk 3]
```diff
@@ -0,0 +1,241 @@
...
+
+.signup-page {
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    z-index: 2;
+    flex-direction: column;
+    position: absolute;
+    width: 35rem;
+    height: 500px;
```
Comment (sri-rishi) [inline index.css]: can use rem here

File: index.css [hunk 4]
```diff
@@ -0,0 +1,241 @@
...
+    left: 60%;
+    top: 25%;
+    gap: 1rem;
+    background: var(--WHITE);
+    opacity: 0.75;
+    box-shadow: 0px 0px 100px rgba(255, 255, 255, 0.25);
```
Comment (sri-rishi) [inline index.css]: You can use just 0 instead of 0px, it will work same

File: index.html [hunk 5]
```diff
@@ -0,0 +1,93 @@
...
+                <h2 class="nav--title">Plan<span style="color: white;">e</span>t</h2>
+            </div>
+            <div class="nav--search">
+                <input type="text" placeholder="Search here" class="nav--search--input">
+                <button class="nav--search--btn">
+                    <i class='fas fa-search' style='color: white'></i>
+                </button>
+
+               
+
```
Comment (sri-rishi) [inline index.html:34]: remove extra spacing in code

File: index.html [hunk 6]
```diff
@@ -0,0 +1,93 @@
...
+        integrity="sha384-DyZ88mC6Up2uqS4h/KRgHuoeGwBcD4Ng9SiP4dIRy0EXTlnuz47vAwmeGwVChigm" crossorigin="anonymous">
+    <title>QUANTUM UI | Home</title>
+</head>
+
+<body>
+
+    <!--navigation--container -->
+
+
+    <div class="nav-bar">
```
Comment (sri-rishi) [inline index.html:22]: for navigation use navbar tag

**Review by sri-rishi — CHANGES_REQUESTED** [review]
Be consistent with units like for units more than 1rem use rems and otherwise use px

**Comment by Adarsh-Patel-Dev** [comment]
> The nav items on right are not displaying to the extreme right on my screen. This is how it's looking.
> 
> Capture11

fixed it

File: index.css (hunk 1 above)
Comment (Adarsh-Patel-Dev) [reply index.css]: done

**Pull request merged**